docs/91_version_history.rst

```diff
@@ -35,229 +35,186 @@ Release Date: 2023-10-17
-       - A security vulnerability in one of the PIP packages used by some of the Python components used by the system.
-       - A numeric list data type has been added to the reading ingestion code of the system.
-       - Audit logs have been add to the user API of the public REST interface.
-       - An issue which could occasionally result in the bearer token used for authentication between the various services expiring before the completion of the renewal process has been resolved. This could result in the failure of services to communicate with each other.
-       - Support has been added to allow filters to ingest pass data onwards during a shutdown of the filter. This allows any buffered data to be flushed to the next filter in the pipeline.
-       - When upgrading the version of a plugin any new configuration items are added to the relevant configuration categories. However the operation was not correctly reported as a configuration change in the audit log. This behaviour has now been corrected.
-       - New public API Entry Points have been added to allow for the creation and manipulation of control pipelines.
-       - The public API that is used to retrieve readings data from the storage layer has been updated to allow data for multiple assets to be retrieved in a single call.
-       - A memory leak in one of the storage plugins has been fixed. This caused the storage service to consume large amounts of memory over time which could result in the operating system killing the service.
-       - The format of Python back traces has been improved to use multiple lines within the log. This makes the trace easier to understand and prevents the truncation that can occur.
-       - The setting of log levels fro a service is now also reflected in any Python code loaded by the service.
+       - Audit logs have been added to the user API of the public REST interface.
        - The plugin developers guide has been updated to include the mechanism for adding audit trail entries from C++ plugins.
        - Plugins that run within the south and north services and north tasks now have access to the audit logging system.
-       - The documentation regarding handling and updating certificates used for authentication has been updated. 
-       - A number of changes have been made to improve the performance of sending data north from the system.
-       - The performance of the statistics history task has been improved. It now makes fewer calls to the storage subsystem, improving the overall system performance.
-       - Changes have been made to the purge process in the sqlitelb and sqliteinmemory plugins in order to improve performance.
+       - The public API has been updated to include the ability to make control requests via the public API.
```

Comment:
"The public API" appears at both the beginning and end of this sentence. It would be sufficient to end the sentence at "... to make control requests."